app/cdap/components/PipelineDiff/DiffWindow.tsx

```diff
@@ -16,36 +16,47 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import { WrapperCanvas } from 'components/hydrator/components/Canvas';
 import { DiffAccordion } from './DiffAccordion';
+import { DiffCanvasWrapper } from './DiffCanvas';
+import { useAppSelector } from './store/hooks';
+import { RootState } from './store';
 
 const DiffWindowRoot = styled.div`
   flex-grow: 1;
   display: flex;
   flex-direction: column;
   height: 100%;
+  background-color: #f6f6f6;
```

Comment:
Please try to use standard colors. You can see an example usage [here](https://github.com/cdapio/cdap-ui/blob/feature/wrangler-2022/app/cdap/components/WranglerV2/DataQualityCircularProgressBar/index.tsx#L32C36-L32C36). We use Mui colors and our own palette [here](https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/components/ThemeWrapper/colors.ts).

Reply:
Done.